### PR TITLE
small fixes for #187

### PR DESCRIPTION
**Pull request by aayusharora**

### CODE REVIEW

File: src/appComponents/twitterwall/singletweet.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ function tweet($timeout,$rootScope) {
     }
     function changeTweet() {
 
-      if (scope.tweetNumber !=== 90) {
+      if (scope.tweetNumber !== 90) {
             scope.tweetNumber = scope.tweetNumber + 1;
             scope.selectedid  =scope.array[scope.tweetNumber].id_str;
```
Comment (juslee) [inline src/appComponents/twitterwall/singletweet.js]: Codacy Issue found: [Multiple spaces found before '='.](https://www.codacy.com/app/triplez/open-event-webapp/file/2532205002/issues/source?bid=3233857&fileBranchId=3284227#l60)

Comment (juslee) [inline src/appComponents/twitterwall/singletweet.js]: Codacy Issue found: [Infix operators must be spaced.](https://www.codacy.com/app/triplez/open-event-webapp/file/2532205002/issues/source?bid=3233857&fileBranchId=3284227#l60)

Comment (juslee) [inline src/appComponents/twitterwall/singletweet.js]: Codacy Issue found: [Expected indentation of 10 space characters but found 12.](https://www.codacy.com/app/triplez/open-event-webapp/file/2532205002/issues/source?bid=3233857&fileBranchId=3284227#l60)

Comment (juslee) [inline src/appComponents/twitterwall/singletweet.js]: Codacy Issue found: [Identifier 'id_str' is not in camel case.](https://www.codacy.com/app/triplez/open-event-webapp/file/2532205002/issues/source?bid=3233857&fileBranchId=3284227#l60)

**Comment by oren** [comment]
:+1:  @aayusharora

**Pull request merged**